sql/core/src/main/scala/org/apache/spark/sql/catalyst/analysis/ResolveSessionCatalog.scala

```diff
@@ -415,6 +404,10 @@ class ResolveSessionCatalog(
         partition)
 
     case ShowCreateTableStatement(tbl, asSerde) if !asSerde =>
+      if (isTempView(tbl)) {
+        throw new AnalysisException(
```

Comment:
actually, can we just call `parseTempViewOrV1Table` here?

Reply:
OK.

That was my original implementation, but I did it this way because you will get `SHOW CREATE TABLE is only supported with temp views or v1 tables` then if you pass temp view, you get `SHOW CREATE TABLE is not supported on a temporary view`.

But I will just call `parseTempVieworV1Table` here. Thanks!